presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcTableHandle.java

```diff
@@ -45,6 +46,10 @@ public JdbcTableHandle(SchemaTableName schemaTableName, @Nullable String catalog
         this(schemaTableName, catalogName, schemaName, tableName, TupleDomain.all(), OptionalLong.empty());
     }
 
+    /**
+     * @deprecated This constructor is intended to be used by JSON deserialization only. Use {@link #builder()} instead.
+     */
+    @Deprecated
```

Comment:
Was your intention to deprecate the short `JdbcTableHandle()` version too?

Reply:
Also if we decide to use long version, it shouldn't be deprecated.

It will need to get deprecated (and removed) when we introduce more pushdown in jdbc.